src/Axiom/Set/Map.agda

```diff
@@ -58,7 +58,7 @@ record IsLeftUnique (R : Rel A B) : Type where
   field isLeftUnique : left-unique R
 
 instance
-  ∅-left-unique : IsLeftUnique {A = A} {B = B} ∅
+  ∅-left-unique : IsLeftUnique {A = A} {B = B} ∅ˢ
```

Comment:
It should be okay to leave superscripts on the emptyset symbol in `.agda` files, since they won't appear in the pdf.

Reply:
Also, in some cases---namely, in the submodules of `Axiom.Set`---we *must* use superscripts; we can't use the type classes there because that causes a cyclic module dependency since the type class imports `Axiom.Set` and `Axiom.Map`.